src/mbgl/tile/tile.hpp

```diff
@@ -25,8 +27,22 @@ namespace style {
 class Layer;
 } // namespace style
 
+using TilePoint = Point<int16_t>;
```

Comment:
We have a typedef for this already, currently spelled `GeometryCoordinate`. I think `TilePoint` is better, but let's wait to introduce it until we can do a universal rename and at the same time remove `GeometryCoordinates` and `GeometryCollection` in favor of geometry.hpp types.